discourse-mixin/dashboards/discourse-jobs.libsonnet

```diff
@@ -0,0 +1,621 @@
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local dashboardUid = 'discourse-jobs';
+
+local prometheus = grafana.prometheus;
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local sidkiqJobDurationPanel = {
```

Comment:
Nitpick: `sidkiq` is not a great abbreviation for Sidekiq. Maybe `sk`, `skiq`, or even just writing it out in full would be more appropriate

Reply:
Renamed to be slightly more condensed 👍